packages/common/src/design-language/theme.ts

```diff
@@ -17,8 +17,9 @@ const theme = {
   // 7 - 32
   // 8 - 36
   // 9 - 40
-  space: [0, 4, 8, 12, 16, 20, 24, 28, 32, 36, 40],
-  sizes: [0, 4, 8, 12, 16, 20, 24, 28, 32, 36, 40],
+  // 10 - 44
```

Comment:
@SaraVieira fyi: the scale in comments is incorrect
0 * 4 = 0
1 * 4 = 4
2 * 4 = 8
3 * 4 = 12
...
10 * 4 = 40
11 * 4 = 44

Reply:
Fixedf and pushged :)